perceval/backends/core/jira.py

```diff
@@ -216,6 +216,22 @@ def metadata_category(item):
         """
         return CATEGORY_ISSUE
 
+    @staticmethod
+    def search_terms(item):
+        """Set the search terms of a Jira items."""
+
+        project_id = item['fields']['project']['id']
+        project_key = item['fields']['project']['key']
+        project_name = item['fields']['project']['name']
+
+        terms = {
+            'project_id': project_id,
```

Comment:
Does make sense to include the `id` of the item too? The one returned by `metadata_id()`.